src/View/SSTemplateParser.peg

```diff
@@ -286,15 +277,17 @@ class SSTemplateParser extends Parser implements TemplateParser
 
         if (isset($sub['Call']['CallArguments']) && isset($sub['Call']['CallArguments']['php'])) {
             $arguments = $sub['Call']['CallArguments']['php'];
-            $res['php'] .= "->$method('$property', [$arguments], true)";
+            $type = ViewLayerData::TYPE_METHOD;
+            $res['php'] .= "->$method('$property', [$arguments], '$type')";
```

Comment:
`true` was for `$cached` - which was just _always_ true so didn't need to be passed in.

`$type` is for "what type of call is this, a property or a method?". I got that idea from twig which does this same thing. It allows to explicitly call methods, or explicitly fetch properties. We don't currently use it as you'll see in `ViewLayerData`, but the option is there for us to play with an introduce new functionality in minor releases if we want, as well as for projects or modules to choose to be more strict about how their view layer accesses data.